### PR TITLE
Add a banner header (props to JoshuaM)

### PR DESCRIPTION
**Pull request by talvasconcelos**
Prepare stall page to show a banner image based on a config file

Draft as it needs a way to pull in a config from another repo, which is not created yet! @arcbtc can you clarify on this?

I see the repo json as :
```
[
    {
        "pubkey": <String: Merchant pubkey>,
        "banner": <String: URL to a banner image (250px height)>,
        "theme": <String: name of the default theme>
    }
]
```

### CODE REVIEW

**Comment by arcbtc** [comment]
Should be more

```
[
  {
    "pubkeys": [
      "<String: Merchant pubkey>",
      "<String: Merchant pubkey>"
    ],
    "banner": "<String: URL to a banner image (250px height)>",
    "primary_color": "<String: hex of color>",
    "secondary_color": "<String: hex of color>"
  }
]
```

**Comment by talvasconcelos** [comment]
> Should be more,
> 
> ```
> [
>   {
>     "pubkeys": [
>       "<String: Merchant pubkey>",
>       "<String: Merchant pubkey>"
>     ],
>     "banner": "<String: URL to a banner image (250px height)>",
>     "primary_color": "<String: hex of color>",
>     "secondary_color": "<String: hex of color>"
>   }
> ]
> ```
> 
> As the most useful feature is people being able to pr their pubkeys to be added to the store.

Cool. 
How does this information gets passed to the market client? a query param? `/?config=<URL to config repo>`?
And where should it live?

**Comment by arcbtc** [comment]
Query param is nice, could also be in here under Merchant, as "Add merchant manifest" 
image

**Comment by arcbtc** [comment]
Under Merchant config I think, if multiple manifests are added it could randomly pick header/theme

**Comment by motorina0** [comment]
Suggestion:
- use `nostr` events for storage and not GitHub or other centralised services
- use the `kind:0` or add a new `kind:3019` for marketplace profile
- the content of the event is everything that a marketplace client needs for it **content** and **look&feel**

eg:
```json
{
    "name": <username>,
    "merchants": <string list: of merchant public keys>,
    "relays": <list of recommended relays>,
    "about": <string>,
    "picture": <url, string: the logo of the marketplace>,
    "banner": <string: URL to a banner image (250px height)>,
    "theme": <string: name of the default theme>,
   ...
}
```

 - then the marketplace `npub` can be used as a query param and/or entered on the client UI

**Comment by talvasconcelos** [comment]
We should probably use naddr, as described in NIP19 for the config event.

https://github.com/nostr-protocol/nips/blob/master/19.md#shareable-identifiers-with-extra-metadata

File: static/components/customer-stall/customer-stall.html [hunk 1]
```diff
@@ -1,79 +1,35 @@
...
-        label="Market"
-        icon="home"
-        @click="$emit('change-page', 'market')"
-        style="cursor: pointer"
-      ></q-breadcrumbs-el>
-      <q-breadcrumbs-el
-        :label="stall?.name || 'Stall'"
-        icon="widgets"
-      ></q-breadcrumbs-el>
+      <q-breadcrumbs-el label="Market" icon="home" @click="$emit('change-page', 'market')"
```
Comment (motorina0) [inline static/components/customer-stall/customer-stall.html:4]: - shouldn't the label dynamic be like in `customer-market`?
`<q-breadcrumbs-el :label="styles.name ?? 'Market'"`

**Comment by motorina0** [comment]
LGTM

In order to complete this PR please:
 - add description of how it can be used
 - open a PR against [NIP15](https://github.com/nostr-protocol/nips/blob/master/15.md) for documenting the event type

File: static/components/customer-stall/customer-stall.html (hunk 1 above)
Comment (talvasconcelos) [reply static/components/customer-stall/customer-stall.html:4]: Yes...

**Pull request merged**